frontend/src/components/auth/Signup/components/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, Center, Text } from "@chakra-ui/react";
+import React from "react";
+
```

Comment:
why is this file called index.tsx?

Reply:
because I kept all the components inside of this index file --> the alternative would be to have each component in its own file and export them out something like this in `index.ts`: `export * as PasswordRequirement from './PasswordRequirement'`
let me know if we want to switch to this
```
> components
---index.ts
----PasswordRequirement.tsx
----MandatoryInputDescription.tsx
----isCheckMarkOrClose.tsx
```

Reply:
i prefer the way u just mentioned!